### PR TITLE
Add Quick Pickup Utils v1.0.0

### PR DESCRIPTION
**Pull request by SimonFlapse**
Quick Pickup Utils G-Earth Extension for Habbo Hotel: Origins

Supports the chat command
`:pickup <wall | floor> <amount>` allowing for versatile pickup of items in your Habbo Hotel: Origins room
Examples:
- `:pickup wall` to pick up all wall items
- `:pickup floor` to pick up all floor items
- `:pickup wall 10` to pick up 10 wall items
- `:pickup floor 5` to pick up 5 floor items
- `:pickup 8` to pickup 8 items starting with wall items and then floor items if there is less than 8 wall items

`:pickup <item_name> <amount>` allowing picking up stacks of the same item in your Habbo Hotel: Origins room
Examples:
- `:pickup rare_daffodil_rug` to pick up all Petals Patches
- `:pickup poster_5000 10` to pick up 10 Purple Garlands

### Additional notes
I've created this plugin primarily as an example project to explore a library for parsing packets from Habbo Hotel: Origins for Java.

Let me know if there is something missing - Thanks!

### CODE REVIEW

File: store/extensions/QuickPickupUtils/extension.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "title": "Quick Pickup Utils",
```
Comment (WiredSpast) [inline store/extensions/QuickPickupUtils/extension.json]: The title and folder name have to be the exact same, you can add spaces in the folder name

File: store/extensions/QuickPickupUtils/extension.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "description": "Chat commands to quickly pickup items in your Habbo flat",
```
Comment (WiredSpast) [inline store/extensions/QuickPickupUtils/extension.json]: I would recommend adding the commands in here, you can use `\n` to add line breaks

File: store/extensions/QuickPickupUtils/extension.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  "source": "https://github.com/SimonFlapse/QuickPickupUtils",
+  "readme": "https://github.com/SimonFlapse/QuickPickupUtils/blob/main/README.md",
+  "releases": "https://github.com/SimonFlapse/QuickPickupUtils/releases",
+
+  "stable": true,
+
+  "framework": {
+    "name": "Native",
+    "version": "1.5.4"
```
Comment (WiredSpast) [inline store/extensions/QuickPickupUtils/extension.json]:
```suggestion
    "version": "1.5.4-beta-6"
```

**Review by WiredSpast — CHANGES_REQUESTED** [review]

File: store/extensions/QuickPickupUtils/extension.json (hunk 2 above)
Comment (WiredSpast) [reply store/extensions/QuickPickupUtils/extension.json]: Most end users won't realize that there is a read me with more info in it

**Comment by SimonFlapse** [comment]
Thanks for the extremely quick review, I'll get it sorted. Thanks ❤️

**Review by WiredSpast — APPROVED** [review]

**Pull request merged**